modulemd/v2/modulemd-module.c

```diff
@@ -293,3 +309,39 @@ modulemd_module_get_stream_by_NSVC (ModulemdModule *self,
 
   return NULL;
 }
+
+
+void
+modulemd_module_add_translation (ModulemdModule *self,
+                                 ModulemdTranslation *translation)
+{
+  gsize i;
+  ModulemdModuleStream *stream = NULL;
+
+  if (!g_str_equal (modulemd_translation_get_module_name (translation),
+                    modulemd_module_get_module_name (self)))
```

Comment:
This should be `g_return_if_fail()` so that it generates a log message and calls `abort()` when `G_DEBUG=fatal-warnings`. We should be able to assume that the caller should already have validated this and isn't just trying to assign translations at random. Or if they are, they should rightly get called out on it.

Reply:
You marked this "resolved" but didn't change it to `g_return_if_fail()` here. I'm guessing this was an oversight.